packages/xpetra/example/Simple/Laplace1D-Tpetra.cpp

```diff
@@ -66,37 +65,35 @@ int main (int argc, char *argv[])
   bool success = false;
   bool verbose = false;
   try {
-    const GO numGlobalElements = 256; // problem size
-    const GO indexBase = 0;
-    RCP<const Teuchos::Comm<int> > comm = Teuchos::DefaultComm<int>::getComm ();
-    RCP<const map_type> map = rcp (new map_type (numGlobalElements, indexBase, comm));
+    const GO numGlobalElements          = 256;  // problem size
+    const GO indexBase                  = 0;
+    RCP<const Teuchos::Comm<int> > comm = Teuchos::DefaultComm<int>::getComm();
+    RCP<const map_type> map             = rcp(new map_type(numGlobalElements, indexBase, comm));
```

Comment:
I personally do not like to insert spaces just to vertically align the equal signs.... but just personal taste....

Reply:
I wouldn't do that either, but I set up clang-format to fix the formatting before I commit.

Reply:
Sure, I am aware of that.